lnd_grpc/lnd_grpc.py

```diff
@@ -106,6 +108,10 @@ def macaroon(self):
     def metadata_callback(self, context, callback):
         callback([('macaroon', self.macaroon)], None)
 
+    def connectivity_event_logger(self, channel_connectivity):
```

Comment:
Callback which logs connectivity changes to self.connection_status automatically